sonar-scanner-engine/src/main/java/org/sonar/scanner/genericcoverage/GenericTestExecutionSensor.java

```diff
@@ -65,21 +65,24 @@ public GenericTestExecutionSensor(TestPlanBuilder testPlanBuilder) {
 
   @Override
   public void describe(SensorDescriptor descriptor) {
-    descriptor.name("Generic Tests Excution Report")
+    descriptor.name("Generic Test Executions Report")
       .requireProperty(REPORT_PATHS_PROPERTY_KEY);
   }
 
   @Override
   public void execute(SensorContext context) {
+    if (context.settings().hasKey(OLD_UNIT_TEST_REPORT_PATHS_PROPERTY_KEY)) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "OLD_UNIT_TEST_REPORT_PATHS_PROPERTY_KEY"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)